src/frontend.jl

```diff
@@ -48,6 +48,13 @@ other environments.
 plotsize() = Atom.plotsize()
 
 """
+    showprofile()
+
+Show currently collected profile information in the editor.
+"""
+showprofile() = isactive() && Atom.Profiler.profiler()
```

Comment:
Why not just `Juno.profile()`? Could also have `profiletree()` for the tree itself, which is actually pretty useful on its own.